.copyright-overrides.yml

```diff
@@ -160,6 +160,8 @@ github.com/dgraph-io/badger:
  - "Copyright 2020 Dgraph Labs, Inc. and Contributors"
  - "Copyright 2021 Dgraph Labs, Inc. and Contributors"
 github.com/mostynb/go-grpc-compression: "Copyright 2023 Mostyn Bramley-Moore."
+github.com/pjbgf/sha1cd: "Copyright 2009 The Go Authors"
```

Comment:
Shouldn't this mention Paulo Gomes in addition to the Go authors?

Reply:
Added. Looks like #21864 added this override yesterday, so I just added Paulo Gomes in that entry